examples/websocket-gemini.yml

```diff
@@ -111,7 +111,9 @@ components:
       payload:
         $ref: '#/components/schemas/market'
       examples:
-        - payload:
+        - name: updateMessage
+          summary: Example of an update message hat contains a change in price information.
```

Comment:
Example of an update message **hat** 

I know that my english isn't good, but `message hat` Wat?! 😄

Reply:
yeah, it is another word, like a synonym for `envelope` (you need to read more about CloudEvents). But yeah, I will switch to `that` this time 😆

Reply:
Thanks for clarification!